src/sql/parts/connection/common/connectionManagementService.ts

```diff
@@ -148,6 +148,16 @@ export class ConnectionManagementService implements IConnectionManagementService
 
 		this.disposables.push(this._onAddConnectionProfile);
 		this.disposables.push(this._onDeleteConnectionProfile);
+
+		// Refresh editor titles when connections start/end/change to ensure tabs are colored correctly
+		let refreshEditorTitlesFunction = () => {
```

Comment:
should we have a hook to update the tab colors when the user edits the server group color, or is that a case we're not too worried about?

Reply:
Good idea, I'll take a look at doing that